core/src/main/java/org/bitcoinj/core/TransactionOutPoint.java

```diff
@@ -226,30 +226,10 @@ public Sha256Hash getHash() {
         return hash;
     }
 
-    /**
-     * @param hash new hash
-     * @deprecated Don't mutate this class -- create a new instance instead.
-     */
-    @Deprecated
-    void setHash(Sha256Hash hash) {
-        this.hash = hash;
-    }
-
     public long getIndex() {
```

Comment:
I think we should at the same time change the accessor names (probably providing deprecations).

Reply:
Fixed and force-pushed.